interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: In JS 'this' is used to access class properties in React.
 
-  Researched answer:
+  Researched answer: This is a JS keyword that refers to the to the object it coresponds to.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is JS library with built in modules for creating user interfaces.
 
-  Researched answer:
+  Researched answer: React is a JS library for building UI and is componenet based.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: I will have to look this up. 
 
-  Researched answer:
+  Researched answer: In a React class component the required lifecycle method is a render method.
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX is a hybrid between Javascript and HTML. One key difference is the use of curly brackets to insert JavaScript 
 
-  Researched answer:
+  Researched answer: JSX is the syntax extension of JavaScript. This allows the developer to directly write HTML in React within JS code. 
 
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: yarn is a package manager that installs node modules when creating a new React application.
 
-  Researched answer:
+  Researched answer: Yarn also provides the following speed,consistency,stability, and security as an alternative to npm (node package manager) that is another popular alternative. 
```

Comment:
Great job on this section!  Some of these answers could be expanded upon a little further like you would do in an interview.